src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -124,8 +124,7 @@ namespace winrt::TerminalApp::implementation
         return appLogic->GetSettings();
     }
 
-    AppLogic::AppLogic() :
-        _reloadState{ std::chrono::milliseconds(100), []() { ApplicationState::SharedInstance().Reload(); } }

```

Comment:
I believe not monitoring `state.json` for changes anymore is important as it prevents disturbing the session state while session persistence is still ongoing. That's because when `ApplicationState` flushes to disk, this FS monitor will be triggered and reload the `ApplicationState` again.

I figured it's not a problem anymore to do this since we're effectively a singleton application now.